solr/core/src/java/org/apache/solr/handler/admin/SecurityConfHandler.java

```diff
@@ -303,11 +306,11 @@ public String toString() {
             }
           });
 
-          this.apis = ImmutableList.copyOf(apis);
+          syncedApis = ImmutableList.copyOf(syncedApis);
```

Comment:
Shouldn't this be assigning to this.apis so that we don't have to do this work and create a new list every time?

Reply:
actually, I guess in the code as it stands the prior assigment will actually allow us to return the prior work, but we haven't assigned the immutable copy anywhere so the next time this is called the return value will be mutable.... If the prior line is removed *then* we repeat the work...